packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -197,6 +197,8 @@ const SwipeableDrawer = React.forwardRef(function SwipeableDrawer(inProps, ref)
           'all',
           getTransitionProps(
             {
+              easing: undefined,
+              style: undefined,
```

Comment:
Indicates that either usage of `getTransitionProps` was the wrong abstraction or that the computed transition is inconsistent with the transition components.